front-end/src/domains/game-view/Game-View.component.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
```

Comment:
This game-view should be in its own directory at the /src level. Reference the architecture guide, and the broader views should be in a /views dir 😎 

https://mammoth-studios.atlassian.net/wiki/spaces/MS/pages/1835256/Frontend+Architecture